docs/formats/README.md

```diff
@@ -266,7 +265,7 @@ and more natural given the super-structure of Zed.  For example, the above
 two SQL-like queries could be written as:
 ```
   count() by shape:=typeof(this)
-  any(this) by shape:=typeof(this) | cut shape
+  any(this) by shape:=typeof(this) | cut any
```

Comment:
Oops... let's get rid of `shape:=` since we are cutting any...

Reply:
Good idea. I pushed a commit to make that adjustment.